### DISCUSSION TITLE
updated yt-dlp and my antivirus software found Win.Virus.Tirusor-9950253-0

### DISCUSSION THREAD

**Discussion by KwisatzJim** [Homebrew/discussions#3282] [Casks]
I updated yt-dlp with 'brew upgrade' and my antivirus software found Win.Virus.Tirusor-9950253-0 in parser.py which is in one of the subdirectories used.  Anyone else seeing this?  is there a proper forum to notify such things?

<img width="1290" alt="Screen Shot 2022-05-18 at 11 03 23 PM" src="https://user-images.githubusercontent.com/131250/169201939-a6174b0a-28f1-446b-a803-c02e7d496dac.png">

yt-dlp: stable 2022.5.18 (bottled), HEAD
Fork of youtube-dl with additional features and fixes
https://github.com/yt-dlp/yt-dlp
/opt/homebrew/Cellar/yt-dlp/2022.5.18 (2,211 files, 24.4MB) *
  Poured from bottle on 2022-05-18 at 17:55:16
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/yt-dlp.rb

**Comment by SMillerDev** [comment]
Homebrew doesn't actually write the software it packages. You should report this to the developers of whatever is triggering the antivirus.
